src/main/java/org/Quotes/Main.java

```diff
@@ -0,0 +1,113 @@
+package org.Quotes;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Main Application
+ */
+public class Main {
+    /**
```

Comment:
Check out the difference between multiline and javadoc comments. This is ok but can cause confusion in case we don't need a javadoc comment above the function definition

Reply:
👍🏻